libs/components/modals/src/lib/modules/modal/modal.component.spec.ts

```diff
@@ -797,6 +809,67 @@ describe('Modal component', () => {
     closeModal(modalInstance);
   }));
 
+  it('should not have the aria-owns property by default', async () => {
+    const modalInstance = openModal(ModalTestComponent, undefined, true);
+    const modalDialogElement = getModalDialogElement();
+
+    expect(modalDialogElement.getAttribute('aria-owns')).toBeNull();
+    await expectAsync(getModalElement()).toBeAccessible();
+
+    closeModal(modalInstance, true);
+  });
+
+  it('should set the aria-owns property to contain the id of the live announce element if it exists', async () => {
+    const liveAnnouncer = TestBed.inject(SkyLiveAnnouncer);
```

Comment:
These tests have similar blocks of code. Factor it out into a function.

Reply:
Done